### PR TITLE
Code migration helper

### PR DESCRIPTION
**Pull request by Spomky**
This PR introduces Rector in order to help migrating and upgrading code using a few set of rules e.g. for PHP or Symfony.

### CODE REVIEW

File: Command/GenerateTokenCommand.php [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 namespace Lexik\Bundle\JWTAuthenticationBundle\Command;
 
+use Traversable;
+use Countable;
+use RuntimeException;
```
Comment (chalasr) [inline Command/GenerateTokenCommand.php]: same CS as Symfony here :) I don't like having use statements for things coming from the global namespace

Comment (Spomky) [reply Command/GenerateTokenCommand.php]: I understand. I removed the rule and reverted the changes

File: Tests/Functional/app/AppKernel.php [hunk 2]
```diff
@@ -37,12 +41,12 @@ public function __construct($environment, $debug, $testCase = null)
...
-            new \Symfony\Bundle\SecurityBundle\SecurityBundle(),
-            new \Lexik\Bundle\JWTAuthenticationBundle\LexikJWTAuthenticationBundle(),
-            new \Lexik\Bundle\JWTAuthenticationBundle\Tests\Functional\Bundle\Bundle(),
-        ];
+        return array(
+            new FrameworkBundle(),
+            new SecurityBundle(),
+            new LexikJWTAuthenticationBundle(),
+            new Bundle(),
+        );
```
Comment (chalasr) [inline Tests/Functional/app/AppKernel.php]:
```suggestion
        return [
            new FrameworkBundle(),
            new SecurityBundle(),
            new LexikJWTAuthenticationBundle(),
            new Bundle(),
        ];
```

**Review by chalasr — APPROVED** [review]
Cool! One minor comment

File: Tests/Functional/app/AppKernel.php (hunk 2 above)
Comment (Spomky) [reply Tests/Functional/app/AppKernel.php]: Strange... Rector reverted to to the old style `array(` for no reason.
It is now fixed.

**Comment by Spomky** [comment]
In the last commit, I took the opportunity to change the `$this->assertSame(xxx, $response->getStatusCode());` into `$this->assertResponseStatusCodeSame(xxx);`

**Comment by chalasr** [comment]
Thank you @Spomky.

**Pull request merged**